docs/README_RECON.md

```diff
@@ -0,0 +1,263 @@
+migration   
+between the source system and databricks system.It supports multiple types of reconciliation depending upon the user   
+requirements and data in the source system.It makes the migration process simpler by removing the complex part of
+validating the data from one system to another and enabling the team with more insights on the data migrated.
+
+## Types of Report Supported:
+
+| report type | description                                                                                                                      | key outputs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+|-------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **schema**  | reconcile the schema of source and target.<br> - are same if the source is same <br> - its compatible if the source is different | - **schema_comparison**<br>- **schema_difference**                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| **row**     | reconcile the data only at row level(hash value of the source row is matched with the hash value of the target)                  | - **missing_in_src**(sample rows in target that are not available in source + sample rows in target that doesn't match with source)<br> - **missing_in_tgt**(sample rows in target that are not available in target + sample rows in source that doesn't match with target)<br>  **NOTE**: the report won't differentiate the mismatch and missing here.                                                                                                                                              |
```

Comment:
Can we add "Preferred when there are no JK identified b/w source and target"

Reply:
done.